DataConnectors/AWS-CloudTrail-Ingestion-Lambda/README.md

```diff
@@ -0,0 +1,118 @@
+# AWS Lambda Function to import CloudTrail Logs to Azure Sentinel
+This Lambda function is designed to ingest AWS CloudTrail Events/S3 Events and send them to Azure Log Analytics workspace using the Log Analytics API.
+
+AWS CloudTrail logs are audit type events from all/any AWS resources in a tenancy. Each AWS resource has a unique set of Request and Response Parameters. Azure Log Analytics has a column per table limit of 500, (plus some system columns) the aggregate of AWS parameter fields will exceed this quickly leading to potential loss of event records
+
+Code does the following things with the logs it processes. 
+1.	Takes the core fields of the record. i.e. all fields except for the Request and Response associated fields and puts them in a LogAnalyticsTableName_ALL. Providing a single table with all records with core event information.	
+2.	Looks at each event and puts it into a table with an extension <AWSREsourceType> i.e. LogAnalyticsTableName_S3 
+3.	Exception to 2 above is for EC2 events, the volume of fields for EC2 Request and Response parameters exceeds 500 columns. EC2 data is split into 3 tables, Header, Request & Response. 
```

Comment:
is this document up-to-date, the multiple behavior is controlled by passed-in parameter instead of the column size. Yes, you may suggest customer pass in different value based on column number, but the code logic doesn't depend on the column number.